### DISCUSSION TITLE
development/2020/06/06/github-action/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#124] [Comments]
#  Github Action 사용법 정리

https://zzsza.github.io/development/2020/06/06/github-action/

**Comment by zzsza** [comment]
<p>좋은글 감사합니다~</p>

>_This comment was made by Disqus. 2021-08-04 09:25:08 JY_

**Comment by zzsza** [comment]
<p>관련 내용이 별로 없어 막막했었는데, 정말 많은 도움이 되었습니다. 감사합니다.</p>

>_This comment was made by Disqus. 2022-01-21 09:35:46 BREW.301_

**Comment by zzsza** [comment]
<blockquote>단, 테스트해보니 약간의 지연 시간이 있음(이건 사용량이 많아서 그럴수도..?)</blockquote><p><br>이 부분에 대해서 Github Action 자체적으로 문제라고 하네요...<br><a href="https://github.community/t/no-assurance-on-scheduled-jobs/133753/2" rel="nofollow noopener" title="https://github.community/t/no-assurance-on-scheduled-jobs/133753/2">https://github.community/t/...</a></p>

>_This comment was made by Disqus. 2022-05-07 03:31:25 방구석호랑이_

**Comment by zzsza** [comment]
<p>좋은 내용 잘 보고 갑니다<br>감사합니다.</p>

>_This comment was made by Disqus. 2022-05-18 02:58:22 denia park_

**Comment by byungchanKo99** [comment]
좋은 글 감사합니다!

**Comment by jwh0124** [comment]
도움이 많이 되었습니다 감사합니다🙏
